VideoEditing/VideoEditor/VideoEditor/Model/VideoBuilder.swift

```diff
@@ -10,11 +10,11 @@ import Foundation
 import AVFoundation
 
 class VideoBuilder{
-
-    var handler: VideoBuilderInteface?
+    
+    var handler: DDVideoHandler? = DDVideoHandler()
```

Comment:
Maybe for testing purposes we should have a screen with two buttons that instantiates the video builder with either yours or Roli class